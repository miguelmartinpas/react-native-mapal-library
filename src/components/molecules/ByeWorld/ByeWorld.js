import React from 'react';
import { View, Text } from 'react-native';

const ByeWorld = () => {
  return (
    <View>
      <Text>Bye World</Text>
    </View>
  );
};

export default ByeWorld;
