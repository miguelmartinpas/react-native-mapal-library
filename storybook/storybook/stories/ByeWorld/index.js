import React from 'react';
import { ByeWorld } from '../../../src';

export default () => {
  return <ByeWorld />;
};
