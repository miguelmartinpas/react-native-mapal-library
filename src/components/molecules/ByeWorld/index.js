import ByeWorld from './ByeWorld';

export default ByeWorld;
