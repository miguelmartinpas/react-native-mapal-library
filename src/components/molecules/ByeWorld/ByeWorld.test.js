import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import ByeWorld from './index';

describe('ByeWorld', () => {
  describe('Rendering', () => {
    it('renders correctly', () => {
      const tree = renderer.create(<ByeWorld />).toJSON();
      expect(tree).toMatchSnapshot();
    });

    it('renders correctly', () => {
      const tree = renderer.create(<ByeWorld />).getInstance();
      expect(tree).toBeDefined();
    });

    it('should match to snapshot', () => {
      const component = shallow(<ByeWorld />);
      expect(component).toMatchSnapshot();
    });
  });
});
