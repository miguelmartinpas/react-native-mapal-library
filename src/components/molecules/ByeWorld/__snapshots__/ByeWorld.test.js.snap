// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ByeWorld Rendering renders correctly 1`] = `
<View>
  <Text>
    Bye World
  </Text>
</View>
`;

exports[`ByeWorld Rendering should match to snapshot 1`] = `ShallowWrapper {}`;
